.travis.yml

```diff
@@ -20,8 +20,8 @@ dist: trusty
 group: edge
 
 env:
-    - PS_VERSION=1.6 DOCKER_IMG=prestashop TAG=1.6 MODULE_TECHNICAL_NAME=None SAUCELABS=None
-    - PS_VERSION=1.7 DOCKER_IMG=prestashop TAG=1.7 MODULE_TECHNICAL_NAME=None SAUCELABS=None
+    - PS_VERSION=1.6 DOCKER_IMG=prestashop TAG=1.6 MODULE_TECHNICAL_NAME=None SAUCELABS=None GDRIVE_REFRESH_TOKEN=1/3XsYauux1bvpC9RE5ap25ffXtQgaSOPz2if9Tt6T9Kw GDRIVE_DIR=0BypobHMgsSrcUFdNWHZLSl9sX28
```

Comment:
`GDRIVE_REFRESH_TOKEN` and `GDRIVE_DIR` to be removed, this information must be private and only available on the test environment.